.github/workflows/main.yml

```diff
@@ -1,30 +0,0 @@
-name: Build
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
-
```

Comment:
This file should not be deleted. Please add this back.

Reply:
Solved